functions.php

```diff
@@ -13,17 +13,17 @@
 if ( ! function_exists( 'twentytwentytwo_support' ) ) :
 
 	/**
-	 * Add support for core block visual styles.
-	 * Styles load in both the editor and the front end.
-	 *
-	 * @link https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#default-block-styles
+	 * Sets up theme defaults and registers support for various WordPress features.
 	 *
 	 * @return void
 	 */
 	function twentytwentytwo_support() {
 
+		// Add support for block styles.
 		add_theme_support( 'wp-block-styles' );
 
+		// Enqueue editor styles.
+		add_editor_style( 'style.css' );
```

Comment:
Is it safer to do this?

```suggestion
		add_editor_style( get_template_directory_uri() . '/style.css' );
```

Reply:
Turns out this doesn't work. 😄 I'll remove in a subsequent commit.